zipline/finance/blotter/core.py

```diff
@@ -0,0 +1,111 @@
+from functools import partial
+from .blotter import Blotter
+from zipline.utils.compat import mappingproxy
+
+
+class BlotterClassDispatcher(object):
+    """
+    A class for registering and dispatching custom blotter classes.
+
+    Method of a global instance of this class are provided by
+    zipline.utils.calendar_utils.
+
+    Parameters
+    ----------
+    blotter_factories : dict[str -> function]
+        Factories for lazy blotter creation.
```

Comment:
Are these actually factories and not just the classes themselves?
This dict should be `dict[str, type]` right?